app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -34,6 +34,9 @@ const getOrbPercent = (endTime, totalTime, offset) => {
   return Math.floor(rangePercent);
 };
 
+const blue = '#6bd9fe';
+const lightBlue = '#8096ff';
```

Comment:
We should keep colors in the theme object. If we need to reference them we should `import` the values.

Here is an example
https://github.com/mojotech/helios2/blob/ffebc0004c15fd93eca79bc6007db6d0c9397229/app/javascript/components/panel-wrapper.jsx#L1-L13

Reply:
https://github.com/mojotech/helios2/pull/272#discussion_r664732264